pom.xml

```diff
@@ -63,7 +62,11 @@
 
         <!-- Test dependencies -->
         <strimzi.version>0.37.0</strimzi.version>
+        <!-- keep Fabric8 version in lock step with the Fabric8 version used by Strimzi -->
         <fabric8.kubernetes.version>6.8.1</fabric8.kubernetes.version>
+        <!-- keep Sundr version in lock step with the Sundr version used by Strimzi and Fabric8 -->
```

Comment:
Where does the dependency on Strimzi come from? I can only see the version being defined but never used...

Reply:
kroxylicious-systemtests pulls in io.strimzi:api

Reply:
Should we be excluding sundr from io.strimzi:api then?